Formula/mysql@5.7.rb

```diff
@@ -149,7 +156,8 @@ def plist
 
     port = free_port
     pid = fork do
-      exec bin/"mysqld", "--bind-address=127.0.0.1", "--datadir=#{dir}", "--port=#{port}"
+      exec bin/"mysqld", "--bind-address=127.0.0.1", "--datadir=#{dir}", "--port=#{port}",
+      "--socket=#{dir}/mysql.sock}"
```

Comment:
```suggestion
      "--socket=#{dir}/mysql.sock"
```

Any ideas why this is needed now but wasn't before?

Reply:
The CI build was failing on one of the target OSXs. I reproduced what I think is the same issue on my iMac by running `brew test mysql@5.7` without this line, and it fails and prints a message about the socket being taken. When I run it with the custom socket, it works.

Reply:
Update: Now that CI has run again, it seems that the socket file wasn't the issue. I haven't been able to reproduce this on my iMac, so will remove this custom socket.